lib/ansible/module_utils/service.py

```diff
@@ -48,8 +48,12 @@ def sysv_is_enabled(name, runlevel=None):
     :kw runlevel: runlevel to check (default: None)
     '''
     if runlevel:
+        if "SUSE" in platform.dist()[0].upper():
+            return bool(glob.glob('/etc/init.d/rc%s.d/S??%s' % (runlevel, name)))
         return bool(glob.glob('/etc/rc%s.d/S??%s' % (runlevel, name)))
     else:
+        if "SUSE" in platform.dist()[0].upper():
+            return bool(glob.glob('/etc/init.d/rc?.d/S??%s' % name))
```

Comment:
actually, for old suse this might not work and others that derive from suse might have same issue, lets use it as a fallback location in general

Reply:
SLES11 is "oldest" supported version. SLES10 has python2.4 - ansible will exit immediately with
```
SyntaxError: future feature absolute_import is not defined
```
Should I change the commit like:
```
if not os.path.isdir('/etc/rc0.d/'):
    return bool(glob.glob('/etc/init.d/rc?.d/S??%s' % name))
```

Reply:
/me  hides his sles8 box 

Still, for other derivatives, it would still be better to not gate it specifically to 'SUSE' dist